core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message.properties

```diff
@@ -0,0 +1,23 @@
+#The MIT License
+#
+#Copyright (c) 2021, CloudBees, Inc.
+#
+#Permission is hereby granted, free of charge, to any person obtaining a copy
+#of this software and associated documentation files (the "Software"), to deal
+#in the Software without restriction, including without limitation the rights
+#to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#copies of the Software, and to permit persons to whom the Software is
+#furnished to do so, subject to the following conditions:
+#
+#The above copyright notice and this permission notice shall be included in
+#all copies or substantial portions of the Software.
+#
+#THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#THE SOFTWARE.
+
+ExecutorsWarning=Distributed builds are set up so consider disabling builds on the controller. See <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">the documentation</a>.
```

Comment:
I am not sure this is the best phrasing we can use, but I don't have suggestions for alternatives.

Strictly speaking, this could just use the same message as the other monitor, emphasizing the potential security problems.

Reply:
I really don't have an opinion on the text. I'm not too fond of the current wording, but don't have much better suggestions at the moment. I have a slight negative opinion on making the text relatively the same between the two.

Reply:
> I have a slight negative opinion on making the text relatively the same between the two.

I mean, it's only separate for internal reasons (to reappear when an instance gets reconfigured to use distributed builds), so in a way it makes sense to have the same message, just slightly different buttons for the next action to take.

IMO reusing the same message would be strictly bitter than this one.

Reply:
This text is indeed confusing. The monitor does not analyze builds at all.
I would recommend writing "The controller allows running jobs on it while there are available nodes or clouds. It might be a security risk. Consider disabling executors on the controller instance"

Reply:
Seems redundant to explain something that the admin is very likely already aware of. I still think reusing the same message from before makes sense here.

(It's basically one monitor, but we want it to appear again once an admin set up their first agent if dismissed before, and this is how we accomplish that.)